vlc-android/src/org/videolan/vlc/viewmodels/MRLPanelModel.kt

```diff
@@ -46,8 +46,7 @@ class MRLPanelModel: ViewModel() {
 
      fun updateHistory() {
           launch(VLCIO) {
```

Comment:
you can execute the whole function in launch(UI) or uiJob, and get history from VLCIO context like this:
```
fun updateHistory() = uiJob {
    val history = withContext(VLCIO) { VLCApplication.getMLInstance().lastStreamsPlayed() }
   observableHistory.value = history
}
```